package.json

```diff
@@ -21,7 +21,7 @@
         "lint:css": "stylelint \"**/*.css\"",
         "lint:js": "eslint ./live-examples/js-examples",
         "start-server": "http-server -p 9090 ./docs",
-        "start-watch": "chokidar '**/{*.js,*.css,*.html,*.json}' -i 'package.json' -i 'docs/**' -i 'node_modules/**' -i 'js/editor-*.js' -c 'npm run build:pages' --initial --silent",
+        "start-watch": "chokidar \"**/{*.js,*.css,*.html,*.json}\" -i \"package.json\" -i \"docs/**\" -i \"node_modules/**\" -i \"js/editor-*.js\" -c \"npm run build:pages\" --initial --silent",
```

Comment:
It feels like this should be inside a `./scripts/` dir because we're doing too much to get this working in the `package.json` but maybe that's good for a follow-up.

Reply:
Actually, how about changing this script to: `chokidar "./live-examples/**/{*.js,*.css,*.html,*.json}" -c "npm run build" --initial --silent`?
Instead of excluding a few files, we could just look for changes in `live-examples`.

Reply:
We could also do that: `chokidar "./live-examples/**" -c "npm run build" --initial --silent`.

Reply:
> We could also do that: `chokidar "./live-examples/**" -c "npm run build" --initial --silent`.

This looks a lot better

Reply:
Great. I have changed the script accordingly.